packages/electron-updater/src/windowsExecutableCodeSignatureVerifier.ts

```diff
@@ -54,8 +54,21 @@ export function verifySignature(publisherNames: Array<string>, unescapedTempUpda
 
           const data = parseOut(Buffer.from(stdout, "base64").toString("utf-8"))
           if (data.Status === 0) {
-            const name = parseDn(data.SignerCertificate.Subject).get("CN")!
-            if (publisherNames.includes(name)) {
+            const subject = parseDn(data.SignerCertificate.Subject)
+            let match = false
+            publisherNames.forEach(name => {
+              const dn = parseDn(name)
+              if (dn.size) {
+                // if we have a full DN, compare all values
+                const allKeys = Array.from(dn.keys())
+                match = allKeys.every(key => {
+                  return dn.get(key) === subject.get(key)
+                })
+              } else if (name === subject.get("CN")!) {
+                match = true
```

Comment:
Can we log a warning here that the developer should provide their full Distinguished Name? Something like this. (Feel free to adjust the message as you see fit)
```
logger.warn(`Signature validated using only CN ${name}. Please add your full Distinguished Name (DN) to publisherNames configuration`)
```